src/components/WorkPoolQueueStatusIcon.vue

```diff
@@ -1,7 +1,18 @@
 <template>
-  <template v-if="workPoolQueue && workQueueStatus">
-    <p-icon :icon="status.icon" size="small" class="work-queue-status-icon" :class="classes" />
-  </template>
+  <p-tooltip
+    v-if="workPoolQueue && workQueueStatus"
+    class="work-queue-status-icon"
+    :text="`Work queue is ${status.state}`"
```

Comment:
On the dashboard it would be useful to include the name in this tooltip. Since there will be multiple work queue statuses grouped together. Maybe a prop to include the name or a slot or something to customize this message would be helpful.